### PR TITLE
chore: add coverage and commitizen

### PR DESCRIPTION
**Pull request by tadjohnston**
[Card](https://rentpath.leankit.com/card/599519508)

changes the way we commit by adding structure

### CODE REVIEW

**Comment by bethstevic** [comment]
Looks good, just a couple of questions. What will the commit structure be and is there a way to override it if we just want to commit a WIP that will be squashed before we push up?

**Comment by tadjohnston** [comment]
@bethstevic great questions! You can still commit the way you were doing this before. Your initial commit should always involve using `git cz`.  We'll be following what commitizen does which is the following:

https://github.com/commitizen/cz-cli#using-the-command-line-tool

File: package.json [hunk 1]
```diff
@@ -19,6 +20,12 @@
     "build-storybook": "build-storybook"
   },
   "jest": {
+    "collectCoverageFrom": [
+      "**/src/**/*.{js}",
```
Comment (lfender6445) [inline package.json]: this covers packages/foo/src?

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,5 @@ storybook-static
 .#*
 packages/**/yarn.lock
 packages/**/package-lock.json
+.coveralls.yml
```
Comment (lfender6445) [inline .gitignore]: did you mean to commit and add to .gitignore?

**Comment by lfender6445** [comment]
should any of these be added to .npmignore?

File: .gitignore (hunk 2 above)
Comment (tadjohnston) [reply .gitignore]: Well.. probably not, but there's no reason to modify it after so we could just leave it as is.

File: package.json (hunk 1 above)
Comment (tadjohnston) [reply package.json]: It collects from src based on the tests.  `**` is an N directories before `/src` so yes it is meant to cover everything in packages. We could add in each repo instead if that would be easier.

**Comment by tadjohnston** [comment]
@lfender6445 good point, we should probably add `coverage` at least to `.npmignore`.

**Comment by grenti** [comment]
This is awesome - LGTM 👍

**Comment by lfender6445** [comment]
👍

**Comment by colinrymer** [comment]
💯 love it!

**Pull request merged**